cl/lib/elasticsearch_utils.py

```diff
@@ -196,9 +289,43 @@ def build_es_main_query(
         build_sort_results(cd),
     )
 
+    search_query = add_es_highlighting(search_query, cd)
+    if cd["type"] == SEARCH_TYPES.ORAL_ARGUMENT:
+        search_query = search_query.sort(build_sort_results(cd))
+
     return search_query, total_query_results, top_hits_limit
 
 
+def add_es_highlighting(search_query, cd):
+    if cd["type"] == SEARCH_TYPES.ORAL_ARGUMENT:
+        ffl = [
```

Comment:
@albertisfu - per our conversation yesterday, I went thru the django-elasticsearch-dsl docs and I don't think they support excluding from source upon mapping creation. You _can_ filter the source like they do here: https://django-elasticsearch-dsl-drf.readthedocs.io/en/latest/source_backend.html

If we want to always exclude certain field manually from source, we could just do that with the Elastic python client here: https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-source-field.html#include-exclude

Reply:
Thanks! Yeah, there is no way to exclude fields on mapping with django-elasticsearch-dsl. However, I found a simpler method to exclude fields from `_source` in the search results. By using `search_query.source(excludes=fields_to_exclude)`, we can achieve the same result without the need to list all the fields we want to include in `_source`.

Regarding your second link on "Including / Excluding fields from _source,"  it is related to the concept of having fields marked as `indexed` and `stored` so that they are not returned by default in `_source`? 
 In our case, we have certain fields that are utilized in the frontend or required in the search API, but they are not used for actual searching or filtering purposes. What do you think might be the appropriate approach for handling this scenario? Additionally, I'm curious if there is any performance difference between having fields defined as "indexed" versus "stored."

If we found that there are advantages to modifying our model mapping fields, we can contemplate using the Elastic Python client instead of relying on `django-elasticsearch-dsl`.